app/mailers/views/school_certifying_officials.html.erb

```diff
@@ -7,7 +7,11 @@
     <p>Dear VA School Certifying Official,</p>
 
     <p>
-      A student enrolled at your school has applied for the Edith Nourse Rogers STEM Scholarship.  The student will also receive a copy of this letter. We can't make a formal determination on the student's eligibility for the STEM Scholarship until we receive the following information from you.
```

Comment:
What happened to the rest of this paragraph?

Reply:
The AC only had one sentence
```In the SCO email, the content

A student enrolled at your school has applied for the Edith Nourse Rogers STEM Scholarship.

is replaced with

A student enrolled at your school or training facility has applied for the Edith Nourse Rogers STEM Scholarship.```

Reply:
I went ahead and added the rest

Reply:
I think you need to check with Darrell, but my understanding is you just replace the sentence, but the rest of the paragraph would remain intact.